frontend/src/utils.ts

```diff
@@ -116,3 +116,23 @@ export function makeListToggler<T>(
     );
   };
 }
+
+export type UserAccountPage = {
+  key: string;
+  userid: string;
+  name: string;
+  surname: string;
+  email: string;
+  currentRole?: string;
+  workspaces: { role: string; name: string }[];
+};
```

Comment:
Doesn't seem the best place in which declaring this...

Reply:
I thought It was the best place since `utils.ts` encloses all the types and functions that are used in other files

Reply:
Utils should include stuff used in different places, this however seems to be more related to that page only, or still, it's where it seems to belong. You can also export it from the component itself if it's required to be used in other places, which also use the component. Still maybe it's not clear to me what is this type for. Can't it be an interface too?

Reply:
Also, what's currentRole for?

Reply:
The type `UserAccountPage` is a simple representation of a user in the UserList component and all the other connected to it (UserListLogic, UploadProgressModal, UserListForm and so on). 

`currentRole` was introduces to have the actual role of a tenant in a specific workspace. This is useful when display the UserList table with the column role